src/frontegg-auth/src/client/tokens.rs

```diff
@@ -75,3 +71,98 @@ pub struct ApiTokenArgs {
 pub struct RefreshToken<'a> {
     pub refresh_token: &'a str,
 }
+
+#[cfg(test)]
+mod tests {
+    use axum::{routing::post, Router};
+    use reqwest::StatusCode;
+    use std::net::{IpAddr, Ipv4Addr, SocketAddr};
+    use std::sync::atomic::{AtomicUsize, Ordering};
+    use std::sync::Arc;
+    use uuid::Uuid;
+
+    use super::ApiTokenResponse;
+    use crate::Client;
+
+    const UNUSED_PORT: u16 = 48557;
+    const ADDR: SocketAddr = SocketAddr::new(IpAddr::V4(Ipv4Addr::LOCALHOST), UNUSED_PORT);
```

Comment:
Use `0` here so it gets dynamically assigned, which may prevent concurrent test run problems.

Reply:
TIL using 0 dynamically assigns the port, thanks!